sqlpage/templates/shell.handlebars

```diff
@@ -43,6 +43,10 @@
         <link rel="alternate" type="application/rss+xml" title="{{title}}" href="{{rss}}">
     {{/if}}
     <meta name="generator" content="SQLPage"/>
+
+    {{#if image_og}}
```

Comment:
Can we rename it to something like `social_image` ? Something that will be easier to understand by someone who hasn't heard of "open graph"